apps/style/common.scss

```diff
@@ -987,10 +1003,6 @@ $workspace-header-button-margin: ($workspace-headers-height - $workspace-header-
   }
 }
 
-#versions-header {
```

Comment:
hm, not sure why this file got autoformatted/wasn't already formatted, but functional changes are here